py/server/deephaven/csv.py

```diff
@@ -75,9 +75,9 @@ def read(
                 dht.byte: _JParsers.BYTE,
                 dht.char: _JParsers.CHAR,
                 dht.short: _JParsers.SHORT,
-                dht.int64: _JParsers.INT,
```

Comment:
This disables support for `dht.int64` and `dht.float64`.  That seems more accurate, but how many users will it affect?

Reply:
`dht.int64 == dht.long`, `dht.float64 == dht.double`; they are present lower down in the map.

Reply:
I'm afraid I don't see dht.int64, dht.float64, dht.int16 or dht.int8 anywhere in this file.  What file is that in?

Reply:
I think I see it.  This is an object map, so you only need one variable that points to the object to be represented.  So in `dtypes.py`, both int64 and long point to the same DType object, so you only need one represented in the map.